src/routes/apiDirectory.js

```diff
@@ -13,15 +14,16 @@ const router = express.Router();
 
 router.use(authMiddleware.unless({ path: [join('/api', loginPath)] }));
 
-router.use('/admin/user', adminRouter);
+router.use('/admin/users', adminRouter);
 
 router.use('/hello', (req, res) => {
   res.send('Hello from ttadp');
 });
 
-router.get('/user', (req, res) => {
-  const { userId, role, name } = req.session;
-  res.send({ userId, role, name });
+router.get('/user', async (req, res) => {
+  const { userId } = req.session;
+  const user = await userById(userId);
```

Comment:
Can we wrap these 2 lines (25-26) into a `try catch` - mainly to prevent calling `toJSON()` if the `user `is `null `for some reason